toolset/setup/linux/frameworks/ffead-cpp-apache-mongo.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+fw_installed ffead-cpp-apache-mongo && return 0
+
+fw_depends mongodb
+fw_depends ffead-cpp-unixodbc
+fw_depends ffead-cpp-mongocdriver
+fw_depends ffead-cpp-httpd
+
+cd ${IROOT}/ffead-cpp-src/
+cp -f web/te-benchmark/sql-src/TeBkWorldmongo.h web/te-benchmark/include/TeBkWorld.h
+cp -f web/te-benchmark/sql-src/TeBkWorldmongo.cpp web/te-benchmark/src/TeBkWorld.cpp
+cp -f web/te-benchmark/config/sdormmongo.xml web/te-benchmark/config/sdorm.xml
+rm -rf ffead-cpp-2.0-bin
+make build-apps
+rm -rf ${IROOT}/ffead-cpp-2.0
+cp -rf ffead-cpp-2.0-bin ${IROOT}/ffead-cpp-2.0
+cd ${IROOT}/ffead-cpp-2.0
+rm -rf web/default web/oauthApp web/flexApp web/markers
+chmod 755 *.sh resources/*.sh rtdcf/autotools/*.sh
+./server.sh > ffead-cpp-apache-mongo.log 2>&1
+while ! echo exit | nc localhost 8080; do sleep 5; done
```

Comment:
ffead-cpp generates a lot of dynamic code on initial startup, which is where it generates and compiles stuff on the fly and creates a dynamic library. This is a one time init step which is very much required.